main.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'app'
+
+# Main
+def main
+  app = App.new
+  loop do
+    app.run
+    puts
+  end
+end
```

Comment:
Your app functionality is perfect! Well done 😄. Please notice that, as requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/211077895-2b640406-a96b-4f34-9d65-79ea697b8a0e.png)

The `main.rb` is the file that is supposed to present the user with the list of options 😉. Please make sure to refactor your code to store the options logic inside `main.rb` instead of inside `app.rb`